Tools/Scripts/libraries/resultsdbpy/resultsdbpy/run-tests

```diff
@@ -30,6 +30,15 @@ import os
 import sys
 import unittest
 
+module_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+tools_dir = os.path.dirname(os.path.dirname(module_dir))
+if os.path.isdir(os.path.join(tools_dir, 'webkitpy')):
+    sys.path.insert(0, tools_dir)
+    import webkitpy
+sys.path.insert(0, module_dir)
+
+import resultsdbpy
```

Comment:
Do we need to invoke autoinstaller somewhere to install cassandra?

Reply:
No, `resultsdbpy` already has the autoinstaller registration configured, we just had not set an autoinstall directory. We don't actually need `webkitpy` to do that, but `webkitpy` centralizes the path we use, which seems like a good thing.